translations/hu/get5.phrases.txt

```diff
@@ -346,7 +346,7 @@
     }
     "MatchPoweredBy"
     {
-        "hu"            "{YELLOW}Get5{NORMAL} által megvalósítva"
+        "hu"            "{YELLOW}Get5 {NORMAL} által megvalósítva"
```

Comment:
You added a space, but you did not move `{NORMAL}` so it has no space after it, which was the whole point. This just has 2 spaces.

Reply:
done